pom.xml

```diff
@@ -146,11 +158,13 @@
             <version>2.2</version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.mockito/mockito-core -->
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
-            <version>2.10.0</version>
+            <version>4.7.0</version>
             <scope>test</scope>
         </dependency>
+
```

Comment:
I don't think there's a need for this new line here. Please look at lines 67 and 68, they do not have any space between them.